cosmoz-omnitable-styles.js

```diff
@@ -83,11 +83,14 @@ export default css`
 		position: relative;
 		overflow: hidden;
 	}
-
 	:host a {
 		color: var(--primary-link-color, inherit);
+		text-decoration: var(--cosmoz-omnitable-link-decoration, none);
```

Comment:
Please keep the underline, as that is the default browser behaviour. We'll style it differently in frontend.
```suggestion
		text-decoration: var(--cosmoz-omnitable-link-decoration, underline);
```